### PR TITLE
feat: Plumbs UpdateActivity and DeleteActivity into the app

### LINKED ISSUE #3874: Update Activity and Delete Activity support
Are there plans to support the [Update Activity and Delete Activity](https://docs.microsoft.com/en-us/azure/bot-service/adaptive-dialog/adaptive-dialog-prebuilt-actions?view=azure-bot-service-4.0#activities) actions? It would be great to be able to e.g. update an adaptive card after a user performed an action.

### PR DESCRIPTION
**Pull request by cwhitten**
closes #3874

note: it still isn't clear where UI is getting component information (label, title, description, etc)

### CODE REVIEW

File: Composer/packages/server/schemas/sdk.uischema [hunk 1]
```diff
@@ -145,6 +145,18 @@
...
+      "label": "Delete activity"
+    }
+  },
+  "Microsoft.DeleteProperty": {
+    "form": {
+      "helpLink": "https://aka.ms/bfc-using-memory",
+      "label": "Delete a property",
+      "subtitle": "Delete Property"
+    }
+  },
```
Comment (a-b-r-o-w-n) [inline Composer/packages/server/schemas/sdk.uischema:159]: This is duplicated.

File: Composer/packages/server/schemas/sdk.uischema [hunk 2]
```diff
@@ -145,6 +145,18 @@
       "subtitle": "Delete Property"
     }
   },
+  "Microsoft.DeleteActivity": {
+    "form": {
+      "label": "Delete activity"
+    }
+  },
```
Comment (a-b-r-o-w-n) [inline Composer/packages/server/schemas/sdk.uischema:152]: Let's make sure this gets added to the SDK component schemas.

File: Composer/packages/ui-plugins/composer/src/defaultMenuSchema.ts [hunk 3]
```diff
@@ -163,4 +163,12 @@ export const DefaultMenuSchema: MenuUISchema = {
     label: formatMessage('Emit a telemetry track event'),
     submenu: [SubmenuNames.Debugging],
   },
+  [SDKKinds.UpdateActivity]: {
+    label: formatMessage('Update activity'),
+    submenu: [SubmenuNames.PropertyManagement],
```
Comment (a-b-r-o-w-n) [inline Composer/packages/ui-plugins/composer/src/defaultMenuSchema.ts:168]: Is this in the right submenu?

Comment (cwhitten) [reply Composer/packages/ui-plugins/composer/src/defaultMenuSchema.ts:168]: There wasn't an exiting menu that felt great. Any ideas?

File: Composer/packages/server/schemas/sdk.uischema (hunk 2 above)
Comment (cwhitten) [reply Composer/packages/server/schemas/sdk.uischema:151]: https://github.com/microsoft/botbuilder-dotnet/issues/4514

**Comment by yeze322** [comment]
@cwhitten we are going to retire the `ConceptLabels` under 'labelMaps.ts' after the UI Schema's Trigger part is finished. Then all kinds of component information could be aggregated together and be much clearer.

**Comment by sschutten** [comment]
How should we use these new actions? We need to get the activity ID from an earlier activity, but the suggested dialog.lastActivityId doesn't exist.

**Pull request merged**